capstone-app/src/app/modules/history/wiki-results.service.ts

```diff
@@ -122,10 +122,11 @@ export class WikiResultsService {
     const regexForExtraColorAttributes = /\d*&.*?;/gi;
     const regexForRefrenceNumbers = /\[[\s\S]+?]/gi;
     const regexForAnyRemainingCSSStyling = /.mw-[\s\S]+?}/gi;
+    const regexNewLine = /(\n)(\n\n)+/gi;
```

Comment:
I never saw any examples that needed a regex like this, It seems like it's just removing the /n you added above?